ui/packages/catalog-ui-search/src/main/webapp/react-component/list-create/list-create.js

```diff
@@ -43,6 +43,7 @@ const AddButton = props => {
       style={{ width: '100%' }}
     >
       <span className="fa fa-plus" />
+      <span> </span>
```

Comment:
❓ why is there an empty span here?

Reply:
(Issue 5) To give the space between the + and the word for these. "+Create New List" and "+Create New List with Result(s)". Original added the space on line 47 to the existing span, but thought this would make it more apparent